### ISSUE TITLE
ApiException is not setting the Errors property for unit tests

### ISSUE THREAD

**Issue by jsgoupil** [square/square-dotnet-sdk#35]
I am trying to create an `ApiException` in Unit Test. I do the following:

```C#
var exception = new Square.Exceptions.ApiException("NotUsed", null);
exception.Errors.Add(new Error("AUTHENTICATION_ERROR", "UNAUTHORIZED"));
```

However, the 2nd line here crashes because you do not initialize the Errors list if I pass in `null` as for the `HttpContext`.
I cannot initialize that variable myself because your `Errors` is `internal`.

I cannot create a mock either, because it doesn't have a parameterless constructor.

Can you please assign an empty list in this case? Unit testing with this `ApiException` is cumbersome otherwise.
I have to use reflection to write to that property...

This is my ugly workaround
```C#
var errors = new List<Error> { new Error("AUTHENTICATION_ERROR", "UNAUTHORIZED") };
var apiException = new Square.Exceptions.ApiException("NotUsed", null);
var propertyInfo = typeof(Square.Exceptions.ApiException).GetProperty(nameof(Square.Exceptions.ApiException.Errors));
propertyInfo.SetValue(apiException, errors);
```

**Comment by sseaman** [comment]
Thanks for pointing this out.  We are tracking your ask and we'll update this issue when we have more progress on it.

**Comment by okenshields** [comment]
This issue has been fixed in release 6.3.0: https://github.com/square/square-dotnet-sdk/blob/master/Square/Exceptions/ApiException.cs#L52
Please let us know if you have further questions/issues. Thanks!
